README.md

```diff
@@ -1,6 +1,6 @@
 <img src="https://www.jisc.ac.uk/sites/all/themes/jisc_clean/img/jisc-logo.svg" align="right" width=50 height=50/><h1 align="left">Octopus</h1>
 
-![ui-tasks](https://github.com/JiscSD/octopus/actions/workflows/ui-tasks.yml/badge.svg)
+![ui-tasks](/actions/workflows/ui-tasks.yml/badge.svg)
```

Comment:
broken on branch but I think will work when merged to main (though for Jisc/octopus it will still be broken or will say 'skipped')